provisioning/device/src/Transports/Amqp/ClientWebSocketTransport.cs

```diff
@@ -88,10 +88,6 @@ private async Task<bool> WriteAsyncCoreAsync(TransportAsyncCallbackArgs args)
             {
                 throw new IOException(webSocketException.Message, webSocketException);
             }
-            catch (HttpListenerException httpListenerException)
```

Comment:
This change is in the AMQP stack, FYI. I don't think we need to remove this catch